spec/controllers/spotlight/resources_controller_spec.rb

```diff
@@ -53,8 +53,7 @@
     describe 'POST create' do
       let(:blacklight_solr) { double }
       it 'create a resource' do
-        allow_any_instance_of(Spotlight::Resource).to receive(:reindex)
-        expect(blacklight_solr).to receive(:commit)
+        expect_any_instance_of(Spotlight::Resource).to receive(:reindex)
```

Comment:
The `POST` here is creating a resource, and one of the side-effects of creating a resource is to trigger (re)indexing.